CMakeLists.txt

```diff
@@ -204,6 +211,7 @@ target_compile_definitions(${PROJECT_NAME}
         -DBOOST_ASIO_STANDALONE
         -DASIO_STANDALONE
         -D$<$<BOOL:${VERBOSE}>:VERBOSE_OUTPUT>
+        -D$<$<BOOL:${PERFORMANCE_TESTS}>:PERFORMANCE_TESTING>
```

Comment:
Could the `-D` parts be removed?

Reply:
Removed, now it is compliant with the Client style.